test/e2e/auth/service_accounts.go

```diff
@@ -235,7 +243,31 @@ var _ = SIGDescribe("ServiceAccounts", func() {
 		})
 	})
 
+	/*
+				Release: v1.9
+				Testname: Service account tokens auto mount optionally
+				Description: Ensure that Service Account keys are  mounted into the Pod only
+		 when AutoMountServiceToken is not set to false. We test the following scenarios here.
+				1. Create Pod,  Pod Spec has  AutomountServiceAccountToken  set to nil
+					a) Service Account is default,
```

Comment:
What does this mean?

Reply:
I meant no changes to out-of-the box behavior without configuration.